types/module/module.go

```diff
@@ -371,9 +384,22 @@ func (m *Manager) InitGenesis(ctx sdk.Context, cdc codec.JSONCodec, genesisData
 			continue
```

Comment:
Can't comment above, but is there any reason not to modify the above function signature to return an error? On my iPad so can't open my laptop

Reply:
I think we might be able to do that, will need to look into it

Reply:
Backporting to 0.47. After that we can refactor a lot of this stuff

Reply:
amazing